dbms/src/Flash/Coprocessor/DAGExpressionAnalyzer.cpp

```diff
@@ -555,13 +555,26 @@ void DAGExpressionAnalyzer::buildCommonAggFunc(
         context);
 }
 
+std::pair<String, DataTypePtr> findFirstRow(
+    const AggregateDescriptions & aggregate_descriptions,
+    const String & arg_name)
+{
+    for (const auto & desc : aggregate_descriptions)
+    {
+        if (desc.function->getName() == "first_row" && desc.argument_names[0] == arg_name)
+            return std::make_pair(desc.column_name, desc.function->getReturnType());
+    }
+    return std::make_pair("", nullptr);
+}
```

Comment:
Please move to `namespace {}` that from L76 to L386

Reply:
done